### DISCUSSION TITLE
[v3] wire:loading doesn't work if wire:target refers to a property in Form

### DISCUSSION THREAD

**Discussion by N1ebieski** [livewire/livewire#6223] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.7

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

1. Create a component with the simple model input:

```php
<?php

namespace App\Livewire\Components;

use Livewire\Component;
use App\Livewire\Forms\TestForm;

class Test extends Component
{
    public TestForm $form;

    public function render()
    {
        return view('livewire.test')->layout('components.layout.app');
    }
}
```

```php
<?php
 
namespace App\Livewire\Forms;

use Livewire\Form;

class TestForm extends Form
{
    public ?string $test = null;
}
```

```html
<div>
    <span 
        class="d-none"
        wire:loading 
        wire:target="form.test"
        wire:loading.class.remove="d-none" 
    >
        Loading...
    </span>
    <input type="text" wire:model.live.debounce.750ms="form.test">
</div>
```

2. Write something in the input. "Loading..." will not show up

3. If you move the property directly to Component class,

```php
class Test extends Component
{
    public ?string $test = null;
}
```

and change the name:

```html
<div>
    <span 
        class="d-none"
        wire:loading 
        wire:target="test"
        wire:loading.class.remove="d-none" 
    >
        Loading...
    </span>
    <input type="text" wire:model.live.debounce.750ms="test">
</div>
```

then "Loading..." state will show properly.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@N1ebieski thanks for reporting! Any chance you could submit a failing test PR? Thanks!

**Comment by uziins** [comment]
I got similiar issue
```
<button wire:loading.attr="disabled" type="submit" class="btn btn-primary">
    <i class="icon-floppy-disk me-2"></i>
    <span wire:loading.remove>Save</span>
    <span wire:loading>Loading...</span>
    </button>
```
On submit, 'Loading..' text appears and 'Save' dissapear, when it's finished only the Loading text returns to its original state, the Save text still dissapear
